e2e/cypress/integration/enterprise/guest_accounts/guest_feature_spec.js

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+const timeouts = require('../../../fixtures/timeouts');
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. #. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @enterprise @guest_account
+
+/**
+ * Note: This test requires Enterprise license to be uploaded
+ */
+
+describe('Guest Accounts', () => {
+    let guestUser;
+
+    before(() => {
+        cy.apiInitSetup();
+
+        cy.apiRequireLicenseForFeature('GuestAccounts');
+
+        // # Log in as a team admin.
+        cy.apiAdminLogin();
```

Comment:
`apiInitSetup` can be safely removed since we're not dealing with team/channel/user. Same with `apiAdminLogin` since up until here, the user session if of sysadmin.
```suggestion
    before(() => {
        cy.apiRequireLicenseForFeature('GuestAccounts');
```

Reply:
Sure, I thought that init setup is mandatory 😅